css/style.css

```diff
@@ -1,3 +1,11 @@
+:root{
+    --color-light-blue: #006DFE;
+    --footer-color: #0E1D28;
+    --button-overlay-color: rgb(136, 11, 22);
+    --header-image-overlay-color: rgba(16, 30, 59, 0.5);
+    --animation-speed: 0.23s;
+}
+
```

Comment:
<img width="596" alt="image" src="https://user-images.githubusercontent.com/52242019/148559487-84a3e600-189c-47ed-a67d-8d1cfa4b60d0.png">
These circles should be pseudoelements.